x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -181,6 +182,16 @@ export const getSavedObjectTypes = (
             },
           ],
         },
+        3: {
```

Comment:
__question:__ @juliaElastic this SO type (`GLOBAL_SETTINGS_SAVED_OBJECT_TYPE`) isn't registered as encrypted saved object, right?

Reply:
Right, it's not an encrypted SO.